style.css

```diff
@@ -169,6 +168,146 @@ body {
 }
 
 /* WORK  SECTION  MAIN AREA */
+
+/* Work Pop up Menu */
+.pop_up_menu {
+  display: flex;
+  padding: 20px 15px 20px 15px;
+  flex-direction: column;
+  justify-content: flex-start;
+  align-items: flex-start;
+  border: 1px solid #ebecf0;
+  background-color: white;
+  box-shadow: 0 48px 48px rgba(37, 47, 137, 0.08);
+  position: fixed;
+  max-height: 100%;
+  top: 40%;
+  left: 40%;
+  transform: translate(-40%, -40%);
+  z-index: 9999;
+  overflow: auto;
+}
```

Comment:
- [ ] As requested by previous reviewer, kindly style your pop up to match the figma template. Kindly add some widht on the mobile design to match template
- [ ] Also center the popup correctly using 
```css
  top: 50%;
  left: 50%;
  transform: translate(-50%, -50%);
```
### Your design
![image](https://github.com/rychrr/portofolio_page/assets/94551484/ac91a7c9-2b89-43f7-9f4c-73ba6fd8df82)
### figma
![image](https://github.com/rychrr/portofolio_page/assets/94551484/cbbb23aa-22a9-42e6-b3fd-377c5061be7a)